resources/splunk_app.rb

```diff
@@ -162,30 +163,40 @@ class PackagedApp < SplunkApp
 
   action :install do
     return unless !version || changed?(:version)
+    @converge_actions = nil # Clear resource updates by property change
```

Comment:
I would be fine putting something along those lines into the code comment.  It's a big comment but I think it's warranted here.